realsense2_camera/config/d455_skynode.yaml

```diff
@@ -0,0 +1,37 @@
+camera1:
+    camera:
+      ros__parameters:
+        device_type: ''
+        serial_no: ''
+        usb_port_id: ''
+
+        config_file: "config/camera-depth-filter-settings.json"
+
+        depth_fps: 10.0
+        depth_width: 848
+        depth_height: 480
+        depth_qos: "SENSOR_DATA"
+
+        infra_fps: 10.0
+        infra_width: 848
+        infra_height: 480
+        infra_qos: "SENSOR_DATA"
+
+        enable_accel: false
+        enable_color: false
+        enable_depth: true
+        enable_gyro: false
+        enable_infra1: true
```

Comment:
do we need infra1? what about infra2? you know the difference?

Reply:
It is the left and the right camera. We do not need any of them, but it comes in handy for testing to have a gray 8bit video stream